microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -598,25 +542,101 @@ See also:
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Extension storage in Application panel
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
+
+_new as of Microsoft Edge Stable 131_
+
+Displays the **Extension storage** section in the **Application** tool.  The **Extension storage** section displays the data that's stored by extensions that are using the storage extension API.
+
+See also: 
+* [chrome.storage](https://developer.chrome.com/docs/extensions/reference/api/storage/) API Reference in the Chrome Extensions docs.
+
+Status:
+* This checkbox is not present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Floating entry points for the AI assistance panel
+
+In Microsoft Edge, this checkbox has no effect.<!-- non-Edge -->
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable experimental performance insights
+
+_new as of Microsoft Edge Canary 133_
+
+Includes experimental insights in the **Insights** side panel in the **Performance** tool.
+
+See also: 
+* [Introduction to the Performance tool](../evaluate-performance/index.md)
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable dimming unrelated events in performance insights and search results
+
+_new as of Microsoft Edge Canary 133_
+
+This experiment changes the color of certain events in the main section of the **Performance** tool to gray.  This happens in two cases:
+
+* When you search for an event by using **Ctrl+F** and then type text, only the search matches retain their coloring; the other events turn gray.
+
+* When you use the **Insights** side-panel, and expand one of the insights, only the events that are related to that insight retain their coloring; the other events turn gray.
+
+See also: 
+* [Introduction to the Performance tool](../evaluate-performance/index.md)
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is not present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable a switch to an alternative timeline navigation option
+
+_new as of Microsoft Edge Canary 133_
+
+In the **Performance** tool, adds a **Switch to an alternative timeline navigation** control inside the **Help/ Keyboard shortcuts** popup.  The switch lets you choose how the main chart reacts to mouse wheel events:
+* The mouse wheel zooms the chart in and out.
+* The mouse wheel pans the chart left and right.
```

Comment:
```suggestion
In the **Performance** tool, adds a **Switch to an alternative timeline navigation** control inside the **Help/ Keyboard shortcuts** popup.
```

I would keep this vague for now, the switch is not even present in DevTools yet, so we don't know exactly how it will work yet.

Reply:
removed detail